routes/api/market.js

```diff
@@ -0,0 +1,85 @@
+const router = require('express').Router();
+const { poster, posterBid } = require('../../controllers/market/poster');
+const { bidUpdate } = require('../../controllers/market/updater');
+const { Market,Bid,Op } = require('../../databases/db');
+
+router.get('/', async(req,res)=>{
+    let {page = 0, size = 10} = req.query;
+    page = parseInt(page);
+    size = parseInt(size);
+    let options = {
+        offset: page * size,
+        limit: size,
+        order: [
+            ['createdAt', 'DESC']
+        ],
+        where: {
+            isFinished: false
+        }
+    };
+
+    const marketActives= await Market.findAll(options);
+    if (marketActives==''){
+        res.json({error: 'No hay ninguna subasta en linea'});
+    }else{
+        res.status(200).json({
+            success: true,
+            items : marketActives
+        });
+    }
```

Comment:
Si no hay ofertas, envien un arreglo vacio no envien "No hay ofertas".
```suggestion
    const marketActives= await Market.findAll(options);
    res.status(200).json({
        success: true,
        items : marketActives
    });
```